src/backend/web/app.js

```diff
@@ -29,8 +30,12 @@ app.set('view engine', 'handlebars');
 app.set('logger', logger);
 app.use(logger);
 
+app.use(express.static(`${__dirname}/static`, { dotfiles: 'allow' }));
```

Comment:
Add a comment above this.  You can use your explanation from the PR about why we need this and what it's used for.